.github/workflows/ci.yml

```diff
@@ -89,10 +89,10 @@ jobs:
       - name: Print station activity
         run: |
           sleep 10 # Wait for Station modules to start
-          docker exec station bin/station.js activity
+          docker logs
```

Comment:
Should this perhaps be `docker logs station` for consistency with the other steps below?